vernac/declare.ml

```diff
@@ -1306,11 +1328,14 @@ let declare_mutual_definition ~pm l =
       Some (List.map3 compute_possible_guardness_evidences wfl fixdefs fixtypes)
     | IsCoFixpoint -> None
   in
+  (* XXX: We should recover the using data here, but we need a test
+     for it first; we stored the using data per-constant, but it is
+     the same for all the constants in a group. *)
```

Comment:
Yes, I don't think it makes sense to have a `using` per-constant of a same block of inductive definitions.